map/gps_track_collection.cpp

```diff
@@ -5,14 +5,10 @@
 #include <algorithm>
 
 using namespace std;
```

Comment:
nit: it would be great to remove global `using namespace std;` from here, as it can break Unity builds (where cpp files are concatenated and using directive unexpectedly spreads between sources).